pyccel/ast/type_inference.py

```diff
@@ -75,8 +81,14 @@ def sp_dtype(expr):
 def str_dtype(dtype):
 
     """
-    return a sympy datatype as string
-    dtype: str, Native Type
+    This function takes a datatype and returns a sympy datatype as a string
```

Comment:
@saidctb Thanks for adding the docstring!
Do we really return the string form of a sympy datatype? Or is it our datatype?

Reply:
a string